src/index.html

```diff
@@ -13,6 +13,45 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__img"
+        src="../src/images/imac.jpeg"
+        alt="apple"
+      />
+
+      <h2 class="card__product-description">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__product-code">Product code: 195434</p>
+
+      <div class="card__stars">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="card__reviews"><p>Reviews: 5</p></div>
+      </div>
+
+      <div class="card__price">
+        <div class="card__price-text"><p>Price:</p></div>
+        <div class="card__price-value"><p>$2,199</p></div>
```

Comment:
This div wrapper if redundant
```suggestion
        <p class="card__price-text">Price:</p>
        <p class="card__price-value">$2,199</p>
```